final-project-plan.md

```diff
@@ -0,0 +1,29 @@
+## Project Plan v1.0
+
+#### Project Name
+Follow the Bucks, an expense tracker
+
+#### Elevator Pitch
+This App is for end-users that would like to monitor their spending. It provides end-users the ability to track their expenses, and compare their spending with budgeted amounts.
+
+#### Wireframe
+Refer to "final-project-wireframe.png" within the pull request.
+
+#### Dependencies
+- **Repository for Expenses**: ideally implemented using Firebase Firestore (with local storage as contingency).
+- **npm modules**: react-fontawesome, prop-types, react-router-dom.
+- **Authentication for App**: ideally implemented using Firebase Authentication (auth not needed with local storage).
+
+#### Task List
+1. Setup project scaffolding
+2. Configure/integrate Firebase Firestore (or local storage)
```

Comment:
Since this will be storing financial data, I'd throw in making sure that your Firestore security rules are all set up too 👍

Reply:
Thanks for the feedback. Good timing, because I just thought about asking you a related question today.

I wanted to clarify the requirement: Your app should be deployed onto a web server and publicly accessible.

I will deploy my app, but I was planning to have it be an app for personal use (single user), so the app would have a sign-in authentication screen but no sign-up functionality. Is this fine??...because I realized belatedly you won't be able to sign-in to the app for grading purposes potentially, though my repo will be public and you'll be able to see it during the presentation :)

Reply:
Yes that'll be fine.

Reply:
Glad to hear. Thanks for the super quick response.